.bundlewatchrc.json

```diff
@@ -1,17 +1,16 @@
 {
-  "trackBranches": ["main"],
+  "ci": {
+    "repoBranchBase": "main",
+    "trackBranches": ["main"]
+  },
   "files": [
-    {
-      "path": "./packages/core/dist/index.js",
-      "maxSize": "200B"
-    },
     {
       "path": "./packages/core/dist/castor.css",
-      "maxSize": "3kB"
+      "maxSize": "5KB"
```

Comment:
Should we watch for a theme as well?

Reply:
Theme consists of theme tokens, same as for Castor Tokens package I don't think we have much control over it.

Reply:
Fair enough.